assignment/scripts/array-practice.js

```diff
@@ -31,14 +31,14 @@ console.log('--- 3. Accessing items in an array ---');
 console.log('First animal is', animalArray[0])
 
 // 3.a. TODO: Log the second animal in the array 
-
+console.log('Second animal is', animalArray[1]);
 
 // 3.b. TODO: Log the last animal in the array using it's array index 
-
+console.log('Last animal is', animalArray[3]);
 
 // 3.c. (STRETCH) TODO: Log the last animal by using the array length, 
 //      instead of the exact index number of the last item
-
+console.log('Last animal is', animalArray.length - 4);
```

Comment:
If you look at the console you will see that this logs _"Last animal is 0"_. Not exactly what the prompt was looking for.